chef/cookbooks/suse-manager-client/recipes/default.rb

```diff
@@ -36,11 +38,10 @@
   EOH
```

Comment:
Well, it initially worked, but I wonder if it still works after update-ca-certificates is called; it sounds to me like the code was working for SLE11 fine, but is not safe for SLE12.